libs/webb-ui-components/src/components/Table/Table.tsx

```diff
@@ -71,7 +70,7 @@ export const Table = <T extends RowData>({
                     }
                   >
                     {header.isPlaceholder ? null : (
-                      <>
+                      <div className="flex items-start justify-start">
```

Comment:
The `className` in the `div` complicates header customization. Consider adding these utilities directly to the `<THeader>` component.